packages/ui-components/src/features/Auth/ForgotPasswordForm.tsx

```diff
@@ -84,7 +86,12 @@ export const ForgotPasswordForm = ({
             Back to log in
           </AuthSubmitButton>
           <AuthLink align="center">
-            Don&rsquo;t have an account? <RouterLink to={registerLink}>Register here</RouterLink>
+            Don&rsquo;t have an account?{' '}
+            {RegisterLinkComponent ? (
+              RegisterLinkComponent
+            ) : (
+              <RouterLink to={registerLink}>Sign up</RouterLink>
+            )}
```

Comment:
This almost certainly fails Prettier formatting, but:

```suggestion
            Don&rsquo;t have an account?{' '}
            {RegisterLinkComponent ?? <RouterLink to={registerLink}>Sign up</RouterLink>}
```